### ISSUE TITLE
🤖 Missing accessibility descriptions on inline actions

### ISSUE THREAD

**Issue by antonis** [wordpress-mobile/WordPress-Android#20212]
The inline actions are missing accessibility descriptions. When being selected in talkback they are announced as "unlabelled"

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by antonis** [comment]
Fixed with https://github.com/wordpress-mobile/WordPress-Android/issues/20212
